.gitignore

```diff
@@ -1,5 +1,6 @@
 configs/config.json
 .idea/
+discord_bot/__pycache__/
```

Comment:
Did some research, turns out these are Python cached bytecode to save "compile" time for unchanged files.

We should instead use `*.pyc` to ignore all files with that cache file type, regardless of folder.

Reply:
Tested - it works
Done